Project.toml

```diff
@@ -1,14 +1,13 @@
 name = "LibCURL"
 uuid = "b27032c2-a3e7-50c8-80cd-2d36dbcbfd21"
-version = "0.5.2"
+version = "0.6.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibCURL_jll = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "0.7, 1.0"
+LibCURL_jll = "7.68"
```

Comment:
Could we bump that up to 7.70 to pick HTTP/2 (`nghttp2`) support as well?

Reply:
Ah yes, I was about to do that.